contracts/koh/contracts/KingAutomaton.sol

```diff
@@ -105,6 +105,12 @@ contract KingAutomaton {
   uint256 constant BITS_PER_VOTE = 2;
   uint256 constant VOTES_PER_WORD = 127;
   uint256 constant BASE_MASK = (1 << BITS_PER_VOTE) - 1;
+  uint256 constant PROPOSAL_START_PERIOD = 90 seconds; // 1 weeks;
+  uint256 constant CONTEST_PERIOD = 90 seconds;  //
+  int256 constant APPROVAL_PERCENTAGE = 50;
```

Comment:
Make APPROVAL_PERCENTAGE / CONTEST_PERCENTAGE parameters of the constructor, so we can test with different values.